tools/meteor-services/deploy.js

```diff
@@ -442,8 +644,20 @@ export function bundleAndDeploy(options) {
     return 1;
   }
 
+  // After an upload succeeds, we want to poll Galaxy to see if the build / deploy succeeds
+  // We used to indicate that the upload was finished via a message to check Galaxy
   if (result.payload.message) {
-    Console.info(result.payload.message);
+    // If there is a newVersionId, then poll for status of the app
+    let newVersionId = result.payload.newVersionId;
+    if(!!newVersionId) {
```

Comment:
Is there a need to cast this to a boolean or can it just be checked as-is?

Reply:
👍